src/components/pages/ParentBooking/BookingCalendar.js

```diff
@@ -0,0 +1,16 @@
+import { Calendar } from 'antd';
+import React from 'react';
+
+const BookingCalendar = () => {
+  const onPanelChange = (value, mode) => {
+    console.log(value.format('YYYY-MM-DD'), mode);
```

Comment:
Wondering what this function is intended to do? Looks like currently, it is just console logging the date?